lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -1364,6 +1366,15 @@ void InterfaceSubContextDelegateImpl::inheritOptionsForBuildingInterface(
       std::string pair = (llvm::Twine(lhs) + "=" + rhs).str();
       GenericArgs.push_back(ArgSaver.save(pair));
   });
+
+  genericSubInvocation.getTypeCheckerOptions().SkipFunctionBodies =
```

Comment:
Probably want to limit this to `All` only. Also need to consider how this impacts the hash - modules without skip/errors should be used by compilations without, but definitely *not* the other way around.